### DISCUSSION TITLE
Team D - Course 3

### DISCUSSION THREAD

**Discussion by 1hk-star** [lablup/contribution-academy-2021#54] [Show and tell]
.

**Comment by youngjun0627** [comment]
### Course 3 : 개발환경 구축하기
210816 이유찬
1. `backend.ai run --rm -c 'print("hello world")' python` 실행하기
![image](https://user-images.githubusercontent.com/57058726/129532213-2abcbcc5-3a78-475d-a044-b6a2b9d1f11b.png)
2. `backend.ai run --rm --exec  'python demo.py' pytorch ./demo.py` 실행하기
![image](https://user-images.githubusercontent.com/57058726/129532693-6425f282-bbe8-4ef4-a177-fc8eeb2e7c7c.png)

210822 이유찬
session mode로 실행하기
![image](https://user-images.githubusercontent.com/57058726/130342575-814faa29-404d-4e2c-9ef3-e83b03cbb0e4.png)

저는 저번에 작성한 파이토치 코드로 실행시켰습니다.
reference : https://github.com/youngjun0627/pytorch_fashionMNIST/blob/main/demo.py

**Comment by JHyunB** [comment]
Course 3 : 개발환경 구축하기
1. backend.ai run python -c "print('Hello World\!')" 실행하기
![helloworld](https://user-images.githubusercontent.com/36959292/130213765-7eb6c9d4-aa1f-457d-84c4-5d5866c1be3a.png)

2. backend.ai run --exec "python ./demo.py" tensorflow ./demo.py 실행하기
![fashion_mnist](https://user-images.githubusercontent.com/36959292/130213825-8adbede6-7e67-4213-9bc5-f68dab8ee4be.png)
예제 코드 실행은 되나, 메모리 부족 문제로 학습결과까지 보지는 못했습니다.

**Comment by 1hk-star** [comment]
1. 
![image](https://user-images.githubusercontent.com/32287560/130324582-4d5a50c3-4cfd-4e1c-8164-c3f526fbf342.png)
2.
